internal/service/streamprivatelinkendpoint/resource_schema.go

```diff
@@ -67,3 +67,8 @@ type TFModel struct {
 	State               types.String `tfsdk:"state"`
 	Vendor              types.String `tfsdk:"vendor"`
 }
+
+type TFModelDSP struct {
+	ProjectId types.String `tfsdk:"project_id"`
+	Results   []TFModel    `tfsdk:"results"`
```

Comment:
nit: i think we normally do `[]*TFModel` but it's just an optimization so it's ok as it is